test-extended.js

```
@@ -0,0 +1,376 @@
+                extractLists: true,
+                extractHeadersFooters: true,
+                generateTailwind: true,
+                generateProseMirror: true
+            }
+        });
+        
+        var result = await instance.convertWithStyles({ path: testFile });
+        
+        log(colors.green, "✓ Style extraction successful");
+        log(colors.cyan, "  HTML length: " + (result.html ? result.html.length : 0) + " characters");
+        log(colors.cyan, "  Paragraph styles: " + Object.keys(result.styles.paragraphStyles || {}).length);
```

Biome (2.3.13)

[error] Shouldn't redeclare 'result'. Consider to delete it or rename it.

'result' is defined here:


(lint/suspicious/noRedeclare)